extension/js/content_scripts/webmail/webmail.ts

```diff
@@ -135,6 +136,7 @@ Catch.try(async () => {
       start,
     });
 
+    BrowserMsg.addListener('pgpMsgType', (r: Bm.PgpMsgType) => PgpMsg.type({ data: Buf.fromRawBytesStr(r.rawBytesStr) }));
```

Comment:
This could be removed, IF pgpMsgType is not used in pgp_block.

Reply:
Done, should `pgpMsgType` be remove from `BrowserMsg` as well?

Reply:
If it's never used, is should be removed